content/en/security_platform/application_security/troubleshooting.md

```diff
@@ -0,0 +1,444 @@
+---
+title: Troubleshooting Application Security Monitoring
+kind: documentation
+further_reading:
+- link: "/security_platform/getting_started/"
+  tag: "Documentation"
+  text: "Get Started Instrumenting Your Application to Send Signals"
+- link: "/security_platform/application_security/"
+  tag: "Documentation"
+  text: "Monitoring Threats with Datadog Application Security"
+- link: "/security_platform/application_security/setup_and_configure/#compatibility"
+  tag: "Documentation"
+  text: "Programming Language and Framework Compatibility"
+---
+
+If you experience unexpected behavior with Datadog Application Security, there are common issues you can investigate, as mentioned below. If you continue to have trouble, reach out to [Datadog support][1] for further assistance. 
+
+## Application Security data is not appearing in Datadog
+
+Check your [tracer startup logs][2]. If `appsec_enabled` is `true`, then Application Security is enabled and running.
+
+Alternatively, you can use the metric `datadog.apm.appsec_host` to check if Application Security is running.
+
+1. Go to **Metrics > Summary** in Datadog.
+2. Search for the metric `datadog.apm.appsec_host`. If the metric doesn’t exist, then there are no services running Application Security. If the metric exists, the services are reported with the metric tags `host` and `service`.
+3. Select the metric, and in the **Tags** section, search for `service` to see which services are running Application Security.
+
+Check for `datadog.trace_agent.appsec.request` metrics to confirm the Datadog Agent is receiving Application Security events.
+
+1. Go to **Metrics > Summary** in Datadog.
+2. Search for `datadog.trace_agent.appsec.request` metrics. If the metrics do not exist, then the Agent is not receiving Application Security events. If the metrics are found, but the Application Security dashboard doesn't show any data, check `datadog.trace_agent.appsec.request_error` to see if there are any errors related to sending the data to the Agent.
```

Comment:
```suggestion
```

All of that is no longer true. It was for our early stage and it is no longer correct.

We should copy here the same troubleshooting metric guides that probably exists in the APM troubleshooting section?

Reply:
Removed this section. Added links to APM troubleshooting.